pkg/cmd/server/server.go

```diff
@@ -446,6 +458,8 @@ func (s *server) validateBackupStorageLocations() error {
 // - Limit ranges go before pods or controllers so pods can use them.
 // - Pods go before controllers so they can be explicitly restored and potentially
 //	 have restic restores run before controllers adopt the pods.
+// - Custom Resource Definitions come before Custom Resource so that they can be
```

Comment:
This has already made it into `master` so can be removed from this PR

Reply:
sorry I was squashing commits for the first time and it squashed everything in between

Reply:
removed